functional_tests/sklearn/plot_clusterer-basic.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+"""Demonstrate basic API of plot_clusterer.
+---
+id: 0.sklearn.plot_clusterer-basic
+plugin:
+    - wandb
+depend:
+    requirements:
+        - numpy
+        - scikit-learn
+assert:
+    - :wandb:runs_len: 1
+    - :wandb:runs[0][exitcode]: 0
+    - :yea:exit: 0
```

Comment:
Please add an assert for the plots.. here is an example for one:
```
assert:
  - :wandb:run[0][summary][elbow_curve][_type]: table-file
  - :wandb:run[0][summary][elbow_curve][ncols]: 3
  - :wandb:run[0][summary][elbow_curve][nrows]: 5
```

similar for silhouette_plot

to figure these out run:
```
yea --debug run myyeafile.py
```

Reply:
done, thanks jeff!